README.md

```diff
@@ -4,6 +4,10 @@
 
 A Terraform module which deploys the requisite micro-services for loading BigQuery on Google running on top of Compute Engine.  If you want to use a custom image for this deployment you will need to ensure it is based on top of Ubuntu 20.04.
 
+## Source acknowledgement
+
+This module was originally sourced from a [community contribution](https://github.com/tnightengale/snowplow-terraform-google-big-query-loader-ce/tree/main?tab=readme-ov-file) by [Teghan Nightengale](https://github.com/tnightengale) in 2022 - big thanks for the help in getting this one started!
```

Comment:
```suggestion
This module was originally sourced from a [community contribution](https://github.com/tnightengale/snowplow-terraform-google-big-query-loader-ce/tree/main?tab=readme-ov-file) by [Teghan Nightengale](https://github.com/tnightengale) in 2022 - big thanks for the help in getting this one started!
```

Reply:
Hey @tnightengale I can't accept the commit without there being an actual change to commit.